### ISSUE TITLE
Break this into a series of small lessons

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/rails-summative-adv-08#15]
Suggested breakdown:
1. Research and user stories, and modeling
2. Create models and controllers
3. Serializers

**Comment by GA-MEB** [comment]
I agree that serializers should probably be spun out, and that there should be a stand-alone lesson for how to do research/user stories/modeling/project planning. However, the explicit intention for this lesson was to walk through every step in the lesson from planning to JS, in order to give students a recipe that they can follow in their own projects.

**Comment by jrhorn424** [comment]
Thanks, I appreciate that intent. However, could we achieve the same intent with smaller lessons, taught in sequence, with a delivered checklist? "Recipes" for development practice should be delivered via checklists.

I know I sound like a broken drum, but checklists save huge cognitive load. (http://www.amazon.com/The-Checklist-Manifesto-Things-Right/dp/0312430000)

**Comment by gaand** [comment]
Should we provide a checklist at/near the end of each unit?

**Comment by jrhorn424** [comment]
If by unit, you mean each materials repo, I'd say yes. They won't all need it. I favor having these lists in the README as opposed separate files. I also favor having them in individual repos over collecting them in meta or somesuch.

**Comment by gaand** [comment]
I thought for the projects.  Checklists for ajax, crud actions, etc.
I think a checklist in the repo probably needs to recapitulate checklists from previous repos.  Until the summative.

**Comment by jrhorn424** [comment]
Hm, if recapitulation is where we want to go, then centrally storing these lists in `meta` and then linking to them will make changes easier.

**Comment by gaand** [comment]
I don't have a strong opinion on recapitulation, just noting it may be necessary.

Attempting content free comment :-P.
